src/Eppie.App/Eppie.App.ViewModels/SettingsPageViewModel.cs

```diff
@@ -30,11 +30,11 @@ namespace Tuvi.App.ViewModels
 {
     public class SettingsPageViewModel : BaseViewModel
     {
-        private bool _showRestartMessage;
-        public bool ShowRestartMessage
+        private bool _isShowRestartMessage;
+        public bool IsShowRestartMessage
```

Comment:
Rename `IsShowRestartMessage` property to `IsRestartMessageVisible` or `IsRestartMessageEnabled`

Reply:
Done.